mypy/checker.py

```diff
@@ -1614,6 +1614,9 @@ def determine_type_of_class_member(self, sym: SymbolTableNode) -> Optional[Type]
         if isinstance(sym.node, TypeInfo):
             # nested class
             return type_object_type(sym.node, self.named_type)
+        if isinstance(sym.node, TypeVarExpr):
+            # No need to check type vars since they can't be used at runtime.
```

Comment:
This comment is a bit unclear. Do you mean that their use is rejected in an expression/runtime context by mypy?

Reply:
Yes.